README.md

```diff
@@ -121,25 +123,36 @@ public function routeNotificationForFacebook()
 - `text('')`: (string) Notification message.
-- `buttons($buttons = [])`: (array) An array of "Call to Action" buttons (Created using `NotificationChannels\Facebook\Attachment\Button::create()`). You can add up to 3 buttons of one of the following types: `web_url`, `postback` or `phone_number`. See Button methods below for more details.
+- `attach($attachment_type, $url)`: (AttachmentType, string) An attachment type (IMAGE, AUDIO, VIDEO, FILE) and the url of this attachment
+- `buttons($buttons = [])`: (array) An array of "Call to Action" buttons (Created using `NotificationChannels\Facebook\Components\Button::create()`). You can add up to 3 buttons of one of the following types: `web_url`, `postback` or `phone_number`. See Button methods below for more details.
+- `cards($cards = [])`: (array) An array of item cards to be displayed in a carousel (Created using `NotificationChannels\Facebook\Components\Card::create()`). You can add up to 10 cards See Card methods below for more details.
 - `notificationType('')`: (string) Push Notification type: `REGULAR` will emit a sound/vibration and a phone notification; `SILENT_PUSH` will just emit a phone notification, `NO_PUSH` will not emit either. You can make use of `NotificationType::REGULAR`, `NotificationType::SILENT_PUSH` and `NotificationType::NO_PUSH` to make it easier to work with the type. This is an optional method, defaults to `REGULAR` type.
 
 ### Available Button methods
 
 - `title('')`: (string) Button Title.
 - `data('')`: (string) Button Data - It can be a web url, postback data or a formated phone number.
-- `type('')`: (string) Button Type - `web_url`, `postback` or `phone_number`.
+- `type('')`: (string) Button Type - `web_url`, `postback` or `phone_number`. Use `ButtonType` enumerator for garunteeing valid values
 - `isTypeWebUrl()`: Helper method to create a `web_url` type button.
 - `isTypePhoneNumber()`: Helper method to create a `phone_number` type button.
 - `isTypePostback()`: Helper method to create a `postback` type button.
 
+### Available Card methods
+
+- `title('')`: (string) Card Title.
+- `subtitle('')`: (string) Card Subtitle.
+- `url('')`: (string) Card Item Url.
+- `image('')`: (string) Card Image Url. Image ratio hould be 1.91:1
+- `buttons($buttons = [])`: (array) An array of "Call to Action" buttons (Created using `NotificationChannels\Facebook\Components\Button::create()`). You can add up to 3 buttons of one of the following types: `web_url`, `postback` or `phone_number`. See Button methods above for more details.
+-
 ## Contributing
 
 Please see [CONTRIBUTING](CONTRIBUTING.md) for details.
 
 ## Credits
 
 - [Syed Irfaq R.](https://github.com/irazasyed)
+- [David Piesse](https://github.com/davidpiesse)
```

Comment:
Contributions from individual contributors are credited in [CHANGELOG](https://github.com/laravel-notification-channels/facebook/blob/master/CHANGELOG.md) and in release notes. We can't really credit each contributor individually in README.

So you can update the [CHANGELOG](https://github.com/laravel-notification-channels/facebook/blob/master/CHANGELOG.md) file with all the changes you've made and either credit yourself or will do it before release.